### ISSUE TITLE
Option to bypass Pomodoro

### ISSUE THREAD

**Issue by ducedo** [toggl/toggl-button#639]
I have some tasks that shouldn't be done in Pomodoro intervals but there's no way around it without going into the settings and enable/disable the option after each task. I tried using Toggl Desktop for non-pomodoro tasks but it syncs directly with the Toggl button which takes over and stops the timer after set minutes.

It would be nice when working with Trello if you had a checkbox or something similar which you could uncheck whenever you did a task that shouldn't be using Pomodoro.

**Comment by IndrekV** [comment]
Actually Toggl Button Pomodoro should only work for entries started from the Toggl Button. Starting entry from anywhere else should no trigger the Pomodoro. Desktop client has it's own pomodoro setting maybe this was what stopped the timer on desktops side.

**Comment by ducedo** [comment]
I've reinstalled both Desktop and Button (at the same time) without solving it. If I close the browser, then Desktop works as it's supposed but not if I have the browser (with Button) open. The Toggl Button starts counting about a second after I active it in Desktop and stops based on the settings in Button.

I even tried having the browser closed while starting the timer in Desktop but when I open the browser, it starts counting as well (same seconds as Desktop) and stops based on its setting. Somehow it overrides the Desktop settings. I've tried reinstalling both and activating/deactivating Pomodoro but I still get the same results.

I'm using OS X / Chrome if it makes a difference.
